integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/generator.py

```diff
@@ -140,6 +138,15 @@ def __init__(
         self.model = model
         self.max_length = max_length
         self.truncate = truncate
+
+        if max_length is not None or truncate is not None:
+            warnings.warn(
+                "The 'max_length' and 'truncate' parameters have been removed and no longer have any effect. "
+                "No truncation will be performed.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
```

Comment:
```suggestion
        if max_length is not None or truncate is not None:
            warnings.warn(
                "The 'max_length' and 'truncate' parameters have been removed and no longer have any effect. "
                "No truncation will be performed.",
            )
```

Technically, this is not just a deprecation, because the parameter no longer has effect. This is an (acceptable) breaking change. So let's modify the warning accordingly to make it more prominent.